docs/howto/operate/setup-ci-cd-aws.md

```diff
@@ -0,0 +1,145 @@
+(operate:setup-ci-cd-aws)=
+# Set up the CI/CD system in an AWS cluster
+
+The pilot-hubs deployer code supports [a new `auth_aws` function](https://github.com/2i2c-org/pilot-hubs/blob/e96e7bcded187870dc2e07d6626de8a12586ed32/deployer/hub.py#L126)
+to get a temporary kubeconfig file that allows a successful connection with the
+kubernetes cluster.
+It is actually expecting a 2i2c generated IAM entity (the `deployer` user) and its
+AWS credentials (that we provide in a sops encrypted json file) to properly set them up
+as the AWS `Access Key ID` and `Secret Access Key` as environment variables and
+successfully perform operations in the cluster as the `deployer` user.
+
+There are some requisites for being able to successfully and automatically deploy hubs
+on AWS clusters using the provided functionality:
+
+1. Find out if you are working with a [AWS EKS-based](https://aws.amazon.com/eks/) or a
+[kops-based](https://kops.sigs.k8s.io/getting_started/aws/) cluster
+  
+   We are currently deploying `EKS`-based (Carbonplan) and `kops`-based (Farallon,
+   Openscapes) clusters. Whether to create one or another depends on the cluster
+   operational needs and some other factors outlined and being discussed in a dedicated
+   [pilot-hub issue](https://github.com/2i2c-org/pilot-hubs/issues/431).
+   You need to know and understand which sort of cluster are you working with to properly
+   set up the config file in step 4.
+
+   Look at [the AWS cluster setup instructions](new-cluster:aws) if you are interested in
+   setting up a `kops`-based cluster.
+
+   :::{note}
+   Documentation about creating an `EKS`-based cluster has not yet being written
+   :::
+
+2. Create a `deployer` user with enough permissions for being able to automatically
```

Comment:
Could this link to an AWS doc about what users are etc if people want to learn more?